app/tests/mem_tests.c

```diff
@@ -177,7 +177,7 @@ static int mem_test(int argc, const console_cmd_args *argv) {
         }
 
         /* allocate a region to test in */
-        status_t err = vmm_alloc_contiguous(vmm_get_kernel_aspace(), "memtest", len, &ptr, 0, 0, ARCH_MMU_FLAG_UNCACHED);
```

Comment:
Unfortunately this and the above code, in lkboot, rely on uncached mappings in order to properly work.